unstructured/partition/pptx.py

```diff
@@ -49,15 +50,17 @@
 @add_chunking_strategy
 def partition_pptx(
     filename: Optional[str] = None,
+    *,
     file: Optional[IO[bytes]] = None,
+    date_from_file_object: bool = False,
+    detect_language_per_element: bool = False,
     include_page_breaks: bool = True,
-    metadata_filename: Optional[str] = None,
-    metadata_last_modified: Optional[str] = None,
     include_slide_notes: bool = False,
     infer_table_structure: bool = True,
     languages: Optional[list[str]] = ["auto"],
-    detect_language_per_element: bool = False,
-    date_from_file_object: bool = False,
+    metadata_filename: Optional[str] = None,
+    metadata_last_modified: Optional[str] = None,
```

Comment:
Adding the "*" just makes explicit that all arguments other than `filename` are keyword-only. This has always been the case because `filename` and `file` can't both be used, but explicit is better than implicit. The remaining line changes are just sorting the kwargs alphabetically for the sake of good order which makes it easier to find the one you're looking for and tell whether one is missing.